websockets/py36/protocol.py

```diff
@@ -0,0 +1,3 @@
+async def __aiter__(self):
+    while True:
+        yield await self.recv()
```

Comment:
Perhaps we should catch an exception and exit cleanly if the other end terminates the connection with a 1000 status code?